components/mailbox/src/Mailbox.svelte

```diff
@@ -711,57 +761,58 @@
               {/each}
             </div>
           {/if}
-          {#if threads.filter((thread) => thread.selected).length}
-            {#if _this.actions_bar.includes(MailboxActions.DELETE)}
-              <div class="delete">
+          {#if _this.actions_bar.includes(MailboxActions.DELETE)}
+            <div class="delete">
+              <button
+                title="Delete selected email(s)"
+                disabled={!threads.filter((thread) => thread.selected).length}
+                aria-label="Delete selected email(s)"
+                on:click={(e) => onDeleteSelected(e)}><TrashIcon /></button
+              >
+            </div>
+          {/if}
+          {#if _this.show_star && _this.actions_bar.includes(MailboxActions.STAR)}
+            <div class="starred">
+              {#each [areAllSelectedStarred ? "Unstar selected email(s)" : "Star selected email(s)"] as starAllTitle}
+                <button
+                  class={areAllSelectedStarred ? "starred" : ""}
+                  title={starAllTitle}
+                  aria-label={starAllTitle}
+                  role="switch"
+                  disabled={!threads.filter((thread) => thread.selected).length}
+                  aria-checked={areAllSelectedStarred}
+                  on:click={(e) => onStarSelected(e)}
+                />
+              {/each}
+            </div>
+          {/if}
+          {#if _this.actions_bar.includes(MailboxActions.UNREAD)}
+            <div class="read-status">
+              {#if !areAllSelectedRead}
                 <button
-                  title="Delete selected email(s)"
-                  aria-label="Delete selected email(s)"
-                  on:click={(e) => onDeleteSelected(e)}><TrashIcon /></button
+                  data-cy="mark-read"
+                  title="Mark selected email(s) as read"
+                  disabled={!threads.filter((thread) => thread.selected).length}
+                  aria-label="Mark selected email(s) as read"
+                  on:click={(e) => onChangeSelectedReadStatus(e)}
```

Comment:
Noticed that this appears to have a different icon when there's a selection than when there isn't. Could we consolidate these?

![image](https://user-images.githubusercontent.com/713991/146558837-53cfae51-e916-4065-b015-5f65a6e7cafa.png)

![image](https://user-images.githubusercontent.com/713991/146558863-8bdad626-aa14-42fb-a309-bc942cdf7e40.png)